test/kafka-ingest-open-loop/mzcompose.py

```diff
@@ -110,6 +110,11 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
     parser.add_argument(
         "--value-bytes", type=int, default=500, help="record payload size in bytes"
     )
+    parser.add_argument(
+        "--upsert",
```

Comment:
a very tiny nit: we could call this flag envelope and have it take the string arguments upsert/none? might be slightly more extensible if we later want to support dbz / dbz upsert as well (i know that we don't yet support avro data in the benchmark and maybe thats feature bloat on the benchmark so feel free to ignore)

Reply:
I thought about this, but decided to keep this simple/straightforward for now. we can always change it later if we need to add more envelopes